### DISCUSSION TITLE
using adapter static and 404 not found on deploy

### DISCUSSION THREAD

**Discussion by aldijokosp** [sveltejs/kit#9260] [Q&A] [answered]
i getting error when deploy using adapter static, when i acces my website with domain www.example.com , it will acces the website, buat if my website access to some specific routes like www.example.com/home, it will show like image on below
<img width="772" alt="image" src="https://user-images.githubusercontent.com/103408450/222080109-77978c48-999c-4e19-bbbb-d2b5e9b5928e.png">

and this for the adapter from svelte.config.js

<img width="378" alt="image" src="https://user-images.githubusercontent.com/103408450/222081066-bc5bb2d9-dd46-4ca4-819a-dae832031555.png">

**Comment by teemingc** [comment, marked as answer]
What kind of hosting are you currently using? Is the webpage found if you append `.html` to the URL?

**Reply by aldijokosp** [reply]
i use a hosting with cpanel, so folder build from sveltekit to public_html on my cpanel

**Reply by teemingc** [reply]
https://kit.svelte.dev/docs/adapter-static#spa-mode-apache

Does adding the following .htaccess rules help?

**Reply by aldijokosp** [reply]
where should me to adding this following? when i click the link, about .htaccess i cant found

**Reply by teemingc** [reply]
Sorry about that. The section was moved recently. This link works https://kit.svelte.dev/docs/single-page-apps#apache

**Reply by aldijokosp** [reply]
thank you for the help, it work for me after change the RewriteRule from 200.html into index.html
